libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -134,8 +134,9 @@ const AP_Param::GroupInfo AP_Airspeed::var_info[] = {
 
     // @Param: _TUBE_ORDER
     // @DisplayName: Control pitot tube order
-    // @Description: Changes the pitot tube order to specify the dynamic pressure side of the sensor. Accepts either if set to 2. Accepts only one side if set to 0 or 1 and can help detect excessive pressure on the static port without indicating positive airspeed.
+    // @Description: This parameter allows you to control whether the order in which the tubes are attached to your pitot tube matters. If you set this to 0 then the first (often the top) connector on the sensor needs to be the stagnation pressure. If set to 1 then the second (often the bottom) connector needs to be the stagnation pressure. If set to 2 (the default) then the airspeed driver will accept either order. The reason you may wish to specify the order is it will allow your airspeed sensor to detect if the aircraft is receiving excessive pressure on the static port compared to the stagnation port such as during a stall, which would otherwise be seen as a positive airspeed.
```

Comment:
this needs to say what stagnation pressure is. 
I suggest to change from:
"needs to be the stagnation pressure" to 
"needs to be the stagnation pressure (the pressure at the tip of the pitot tube)"

we don't want to require users to lookup wikipedia when setting up pitot at the flying field